railgun/config/crd/kustomization.yaml

```diff
@@ -4,6 +4,10 @@
 resources:
 - bases/configuration.konghq.com_tcpingresses.yaml
 - bases/configuration.konghq.com_udpingresses.yaml
+- bases/configuration.konghq.com_kongclusterplugins.yaml
```

Comment:
I believe we have those files under deploy/manifests, deploy/single

Reply:
Can you clarify? What's the requested change or question here?

Reply:
Can you please re-use this yaml https://github.com/Kong/kubernetes-ingress-controller/blob/main/deploy/manifests/base/custom-types.yaml#L5 or remove duplicates ?

Reply:
Similar to the RBAC resources, 2.x CRDs are generated by kubebuilder. We specifically do not want to reuse the 1.x CRDs.